_drafts/makefiles.md

```diff
@@ -0,0 +1,329 @@
+---
+layout: post
+title: Making Makefiles
+subtitle: Learn how to use Makefiles to automate any sort of task, including but
+  not limited to compiling.
+permalink: /making-makefiles/
+---
+
+I'm guessing the other TAs briefly exposed you to `Makefile`s in discussion and
+then left you terribly confused. (Or not. I'll admit that I don't go to their
+discussions, so I wouldn't know. But let's pretend, so that I have something to
+talk about, that you want a refresher on how `Makefile`s work.)
+
+In this document, we'll see how to construct a `Makefile` from the bottom up,
+and have it do useful things like compile our project and make a submit archive.
+On the way there, you'll also get to see me struggling to use the verb "make" so
+as to avoid any confusion with the program.
+
+# What is `make`?
```

Comment:
I have no idea what you're talking about. The raw text? The compiled text? The organization of the writer's ideas? The ideas themselves? Are you upset that in some views on Github but not others it does unfortunate word-wrapping?

![screen shot 2015-08-29 at 5 33 40 pm](https://cloud.githubusercontent.com/assets/454057/9564397/21eae7e4-4e74-11e5-9931-d0e910a53f39.png)